app/src/ui/create-tag/create-tag-dialog.tsx

```diff
@@ -99,9 +114,21 @@ export class CreateTag extends React.Component<
     return null
   }
 
+  private getExistingTagsFiltered(filter: string = ''): Array<string> | null {
+    const previousTags = []
+    if (this.props.localTags) {
+      for (const item of this.props.localTags.keys()) {
+        previousTags.push(item)
+      }
+      return previousTags.filter(item => item.includes(filter))
+    }
+    return null
+  }
+
```

Comment:
We prefer checks + early returns, and also I think you can create an array from `localTags.keys()` just using `Array.from`

```suggestion
  private getExistingTagsFiltered(filter: string = ''): Array<string> | null {
    if (this.props.localTags === null) {
      return null
    }
    const previousTags = Array.from(this.props.localTags.keys())
    return previousTags.filter(item => item.includes(filter))
  }

```